### PR TITLE
fix: renamable page title throwing react error

### LINKED ISSUE #1487: RenamablePageTitle is making state changes after unmounting
```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in RenamablePageTitle (created by VEOHeader)
    in div (created by PageHeader)
    in div (created by PageHeader)
    in PageHeader (created by VEOHeader)
    in VEOHeader (created by EditViewVEO)
```

#### To Fix:
https://dev.to/otamnitram/react-useeffect-cleanup-how-and-when-to-use-it-2hbm

### Reproduction Steps
1. From any dashboard
1. Pick a cell, click the gear icon
1. Edit the cell's title. Click the cell's checkmark
1. The console should throw the above warning

### Acceptance Criteria
- Editing a cell title doesn't throw a react warning

### PR DESCRIPTION
**Pull request by blegesse-git**
Closes #1487 

On saving the updated title, the function was making a component state change after unmounting RenamablePageTitle which was throwing a react error to use cleanUp in useEffect hook. Instead we removed the state change as it is not needed after the component has been unmounted.

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 48 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1511)

File: src/pageLayout/components/RenamablePageTitle.tsx [hunk 1]
```diff
@@ -45,8 +45,6 @@ const RenamablePageTitle: FC<Props> = ({
     if (onClickOutside) {
       onClickOutside(e)
     }
-
-    setEditingState(false)
```
Comment (Asalem1) [inline src/pageLayout/components/RenamablePageTitle.tsx:49]: what happens if `onClickOutside` is undefined?

Comment (hoorayimhelping) [reply src/pageLayout/components/RenamablePageTitle.tsx:49]: good call. we needed to move the `setEditingState` call into the cleanup function in the `useEffect` hook.

**Review by hoorayimhelping — APPROVED** [review]
Great work! Welcome to the codebase, Beth!

**Pull request merged**